c/polyjuice.h

```diff
@@ -491,12 +517,13 @@ void selfdestruct(struct evmc_host_context* context,
   int ret;
   if (memcmp(beneficiary->bytes, context->destination.bytes, 20) == 20) {
```

Comment:
Is the condition right?? memcmp(...) == **20**?

Reply:
Oops.

Reply:
This check is not necessary, remove it.